guillotina/subscribers.py

```diff
@@ -28,3 +30,38 @@ def modified_object(obj, event):
         return []
 
     return await sitemanager.adapters.asubscribers((event.object, event), None)
+
+
+@configure.subscriber(
+    for_=IRequestFinishedEvent)
+async def add_http_cache_headers(event):
+    """This will add, if configured, the corresponding http cache headers
+    on the response of GET requests
+    """
+    if event.request.method != 'GET':
+        # Only cache responses from get requests
+        return
+
+    httpcache_settings = app_settings.get('http_cache', {})
```

Comment:
Use a function to define default cache policy with resource, view  and request as parameters so we can create specific cache policies and merge with view overwrite configuration